src/vsphere_cpi/spec/unit/cloud/vsphere/datastore_picker_spec.rb

```diff
@@ -378,6 +378,10 @@ module VSphereCloud
           sorted_datastores = available_datastores.sort do |x, y|
             y[1].free_space <=> x[1].free_space
           end
+          puts 'Summary of simulated placements'
```

Comment:
I removed this earlier. Why do we need to print this? Is it to verify the randomization taking place?

Reply:
yes, Like earlier it was 400 for each datastore since each disk was going into each datastore. We could think of may be adding a check instead of printing if those no's are different when we refactor